Tests/FluentAssertions.Specs/Types/PropertyInfoSelectorSpecs.cs

```diff
@@ -355,6 +355,45 @@ public void When_selecting_properties_return_types_it_should_return_the_correct_
                 , typeof(string), typeof(int), typeof(int), typeof(int), typeof(int)
             });
     }
+
+    [Fact]
+    public void When_selecting_properties_with_only_set_accessors_it_should_return_the_applicable_properties()
```

Comment:
```suggestion
    public void When_a_property_only_has_a_public_setter_it_should_be_included_in_the_applicable_properties()
```

Reply:
So basically refactoring the `ThatArePublicOrInternal` tests under a public class `ThatArePublicOrInternal` i assume.